bench/runbench.py

```diff
@@ -117,13 +124,19 @@ def build_rho(gitref, args, jit, build=True):
             subprocess.call(['git', 'fetch', 'origin'])
         # Clean and switch to the selected revision.
         subprocess.call(['git', 'reset', '--hard', gitref])
-        # Build with JIT enabled.
-        subprocess.call(['git', 'clean', '-fd'])
-        subprocess.call(['git', 'clean', '-fX'])
         if build:
+            if not args.no_clean:
+                subprocess.call(['git', 'clean', '-fd'])
+                subprocess.call(['git', 'clean', '-fX'])
             subprocess.call(['tools/rsync-recommended'])
             if jit:
-                subprocess.call(['./configure', '--with-blas', '--with-lapack', '--enable-llvm-jit'])
+                # Build with JIT enabled.
+                # Need to use Clang for JIT build.
+                env = os.environ.copy()
+                env['CC'] = 'clang'
+                env['CXX'] = 'clang++'
```

Comment:
I agree. I've updated the PR to set the same environment for both JIT/non-JIT benchmarks.